toxav/toxav.c

```diff
@@ -1086,8 +1075,11 @@ bool toxav_video_send_frame(ToxAV *av, uint32_t friend_number, uint16_t width, u
         memcpy(img.planes[VPX_PLANE_U], u, (width / 2) * (height / 2));
         memcpy(img.planes[VPX_PLANE_V], v, (width / 2) * (height / 2));
 
+        // TODO(zoff99): don't hardcode this, let the application choose it
+        const vpx_enc_deadline_t deadline = VPX_DL_REALTIME;
```

Comment:
@zoff99 should we just remove this line + the todo?

Reply:
> also don't use `0` on vpx_codec_decode i think that was not optimal. but again these things need proper real world testing.

https://github.com/webmproject/libvpx/blob/d18c34c54a1b05797d245367ee5ac05a85cf1ccc/vpx/vpx_decoder.h#L206-L209

Reply:
testing with only 1 client and only on desktop. that's not proper testing and has bitten us in the backside before.

Reply:
You should try the encode change with your clients then. :)